app/domain/operations/individual_market/submit_and_determine_application.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
```

Comment:
@ymhari too your feedback on renaming the submit operation and submitting it as part of this operation.